detekt-config.yml

```diff
@@ -45,6 +45,9 @@ potential-bugs:
     excludes: "**/test/**,**/androidTest/**,**/*.Test.kt,**/*.Spec.kt,**/*.Spek.kt"
   UnsafeCast:
     active: true
+  InvalidRange:
```

Comment:
If you don't mind I'd prefer to annotate the concerned code with `@Suppress("InvalidRange")` rather than changin this file.

Reply:
No worries, fixed it now  👍